### PR TITLE
Enable use of existing org, space and quota (all or nothing) when running the tests instead of generated entities.

### PR DESCRIPTION
**Pull request by coreygibbs**
Allows tests to run using a pre-created organization, space and quota based on settings in the integration.json file.  Some implementations of CF are not ideal for creating new organizations on the fly and this change allows testing under those conditions.

### CODE REVIEW

File: helpers/context.go [hunk 1]
```diff
@@ -144,10 +163,15 @@ func (context *ConfiguredContext) Teardown() {
...
-		if !context.isPersistent {
+		if !context.isPersistentApp && !context.isPersistentOrgAndSpace {
 			runner.NewCmdRunner(cf.Cf("delete-org", "-f", context.organizationName), context.shortTimeout).Run()
 			runner.NewCmdRunner(cf.Cf("delete-quota", "-f", context.quotaDefinitionName), context.shortTimeout).Run()
 		}
+
+		if context.isPersistentOrgAndSpace {
+			runner.NewCmdRunner(cf.Cf("t", "-o", context.organizationName), context.shortTimeout).Run()
+			runner.NewCmdRunner(cf.Cf("delete-space", "-f", context.spaceName), context.shortTimeout).Run()
+		}
```
Comment (zrob) [inline helpers/context.go]: @coreygibbs I'm not sure I understand why you are deleting the space if it is meant to be persistent.  I might be missing something here, could you help explain?

**Comment by zrob** [comment]
Thanks @coreygibbs 

We added a question about space deletion. 

We are also concerned about how this interacts if you use parallel nodes,  that mode typically creates additional orgs, are you just always running everything in serial?

Thanks,
@zrob & @jberkhahn

**Comment by jeffpak93** [comment]
Hi @coreygibbs ,

It seems like there hasn't been much activity on this PR for a while. Are you still interested in pursuing these changes? Feel free to let us know.

Thanks,
@jeffpak93 and @swetharepakula, CF CAPI Team

**Comment by swetharepakula** [comment]
Hi @coreygibbs,

We are closing this issue for now. Please feel free to reopen or file another PR if this is still an issue.

Thanks,
@swethar and @jberkhahn

**Pull request closed without merging**